docs/project/implementations.md

```diff
@@ -4,37 +4,38 @@ title: Who's using it?
 sidebar_label: Who's using it?
 ---
 
-This is a list of projects which have implemented the `all-contributors` specification. To add your project to this
+This is an alphabetical list of projects which have implemented the `all-contributors` specification. To add your project to this
```

Comment:
Not sure if explicitly specifying that it's alphabetical is necessary.

Reply:
I changed it because previously, the list was not alphabetised, so users who have read the message in the past might not notice the change. It's not important, so it can be changed back if you want.

Reply:
Can you order the list alphabetically and case-insensitive?

Reply:
I can, but it would be harder to maintain over time. Computers alphabetise case-sensitively. Of course, I'll change it because you asked.